prv_accountant/privacy_random_variables/laplace_mechanism.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import numpy as np
+
+from .abstract_privacy_random_variable import PrivacyRandomVariable
+
+
+class LaplaceMechanism(PrivacyRandomVariable):
+    def __init__(self, mu: float) -> None:
+        self.mu = mu
+        assert self.mu > 0
+
+    def cdf(self, t):
+        return np.where(
+            t >= self.mu,
+            1,
+            np.where(
+                np.logical_and(-self.mu < t, t < self.mu),
+                1/2*np.exp(1/2*(t-self.mu)),
+                0
+            )
+        )
+
+    def rdp(self, alpha):
+        return 1/(alpha-1) * np.log(
+            alpha/(2*alpha-1) * np.exp((alpha-1)*self.mu) + (alpha-1)/(2*alpha-1) * np.exp(-alpha*self.mu)
+        )
```

Comment:
As mentioned in the notebook: I am not sure if it is needed here, but would a reference to these equations help understanding? I'll leave it up to you.